src/Chapter2.hs

```diff
@@ -740,7 +764,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Eq a => a -> [[a]] -> [[a]]
+contains x l = filter(\list -> x `elem` list) l
```

Comment:
Now, after you've mastered eta-reduction, you can apply such a technique to this function as well  🙂 
But your implementation is already great 💯 
I would only add space between function and argument as this is the standard in Haskell 😉